src/ApiBureau.Bullhorn.Api/Dtos/CandidateBaseDto.cs

```diff
@@ -17,13 +17,12 @@ public class CandidateBaseDto : EntityBaseDto
     public bool IsDeleted { get; set; }
     public string Status { get; set; } = null!;
     public long CustomDate2 { get; set; }
-    public List<string> Source { get; set; }
+    public string Source { get; set; }
```

Comment:
I see. There is some legacy code which doesn't make sense anymore. Thanks for pointing this out. Would you like to update this PR so the Source is nullable according the Bullhorn docs? Or I can do it after the merge.

Obviously many other things should be nullable but as we are fixing this one at least this can be corrected as well :).